frontend/app/src/locales/en.json

```diff
@@ -2717,7 +2723,155 @@
     "transaction": "TX:"
   },
   "transactions": {
-    "clear_selection": "Clear Selection",
+    "actions": {
+      "details": "Details",
+      "redecode_events": "Redecode events",
+      "add_event": "Add a new event"
+    },
+    "details": {
+      "title": "Transaction details",
+      "nonce": "Nonce",
+      "gas_fee": "Gas Fee",
+      "gas_limit": "Gas Limit",
+      "gas_price": "Gas Price",
+      "gas_used": "Gas Used",
+      "input_data": "Input Data",
+      "internal_transaction": "Internal"
+    },
+    "events": {
+      "actions": {
+        "edit": "Edit event",
+        "delete": "Delete event"
+      },
+      "confirmation": {
+        "delete": {
+          "title": "Delete Event",
+          "message": "Are you sure you want to delete the event?",
+          "action": "Confirm"
+        },
+        "ignore": {
+          "title": "Cannot Delete Event",
+          "message": "At least 1 event required for the transaction. Do you want to ignore the transaction in the accounting instead",
+          "action": "Yes, Ignore the Transaction"
+        }
+      },
+      "customized_event": "Customized event",
+      "dialog": {
+        "save": "Save",
+        "add": {
+          "title": "Add Event"
+        },
+        "edit": {
+          "title": "Edit Event"
+        }
+      },
+      "form": {
+        "asset": {
+          "validation": {
+            "non_empty": "The asset field cannot be empty"
+          }
+        },
+        "amount": {
+          "label": "Amount",
+          "validation": {
+            "non_empty": "The amount field cannot be empty."
+          }
+        },
+        "counterparty": {
+          "label": "Counterparty"
+        },
+        "date": {
+          "hint": "The date of when the event happened",
+          "label": "Date"
+        },
+        "event_type": {
+          "label": "Event Type"
+        },
+        "event_subtype": {
+          "label": "Event Subtype"
+        },
+        "location": {
+          "label": "Location",
+          "validation": {
+            "non_empty": "The location field cannot be empty."
+          }
+        },
+        "location_label": {
+          "label": "Location Label"
+        },
+        "notes": {
+          "label": "Notes",
+          "hint": "[Optional] Additional notes to store for the action"
+        },
+        "usd_value": {
+          "label": "USD Value",
+          "validation": {
+            "non_empty": "The usd value field cannot be empty."
+          }
+        },
+        "sequence_index": {
+          "label": "Sequence Index",
+          "validation": {
+            "non_empty": "The sequence index field cannot be empty."
+          }
+        }
+      },
+      "headers": {
+        "type": "Type",
+        "asset": "Asset",
+        "description": "Description"
+      },
+      "history_event_type": {
```

Comment:
This is not going to scale for type and subtype. We need to somehow take these from the backend as they may change at any point.

Is there any test you guys can add that checks them against the backend so we know if we break compatibility?